cfdm/core/meta/docstringrewrite.py

```diff
@@ -517,21 +577,24 @@ class that inherits such such a method will need to exclude it,
 
     @classmethod
     def _docstring_update(cls, package_name, class_name, f,
-                          method_name, config):
+                          method_name, config, class__doc__=None):
```

Comment:
I feel `class__doc__` could be a slightly confusing name for an argument. Though it is very clear what it is referring to, at a glance I personally would see the double underscores and without looking carefully think it was `class.__doc__` when used internally. So, it's not crucial, but if you agreed for our future benefit perhaps renaming to `class_docstring` or similar would be better.